### PR TITLE
added HW5

### PR DESCRIPTION
**Pull request by DinnoAnya**

### CODE REVIEW

**Comment by DinnoAnya** [comment]
Added HW6

File: HalahanAnna/HW/HW5/Task1.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#int_list = [10, 3, 45, 2, 7]
```
Comment (kolyasalubov) [inline HalahanAnna/HW/HW5/Task1.py:1]: good job

File: HalahanAnna/HW/HW3/Task3.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (kolyasalubov) [inline HalahanAnna/HW/HW3/Task3.py:1]: a, b = b, a

File: HalahanAnna/HW/HW5/Task2.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+n = int(input("print numbers: "))
```
Comment (kolyasalubov) [inline HalahanAnna/HW/HW5/Task2.py:1]: good job
I recommend using more meaningful variable names

File: HalahanAnna/HW/HW6/Task1.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+numb_two = [num for num in range(1, 11) if num % 2 == 0]
```
Comment (kolyasalubov) [inline HalahanAnna/HW/HW6/Task1.py:1]: good job

File: HalahanAnna/HW/HW6/Task2.py [hunk 5]
```diff
@@ -0,0 +1,11 @@
+correct_log= "First"
```
Comment (kolyasalubov) [inline HalahanAnna/HW/HW6/Task2.py:1]: good job

**Review by kolyasalubov — APPROVED** [review]

**Pull request merged**